ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/event-handlers.j2

```diff
@@ -59,23 +59,27 @@
 {%             elif handler.trigger_on_maintenance.action is arista.avd.defined("end") %}
 {%                 set trigger_config = trigger_config ~ " end" %}
 {%             endif %}
-{%         elif handler.trigger is arista.avd.defined("on-counters") %}
+{%         elif handler.trigger is arista.avd.defined("on-counters") and handler.trigger_on_counters is arista.avd.defined %}
```

Comment:
can you fix the blank space on (https://github.com/aristanetworks/avd/pull/4055/files#diff-88c0e2b99b06690b2fa1daa904575a2cadf99edb621368705900d16a3e9545b5L42-R42).

Not sure why j2lint not captured unwanted space

Reply:
Fixed